### ISSUE TITLE
PostCSS does not work in npm run build

### ISSUE THREAD

**Issue by ianbelanger79** [WordPress/twentytwenty#810]
It seems that the recent changes to the build tools do not add vendor prefixes. The RTL stuff still works @ntwb

**Comment by nielslange** [comment]
Did you run `npm install` before running `npm run build`, @ianbelanger79?

**Comment by nielslange** [comment]
That's what I see in my terminal when running `npm run build` after running `npm install` before:

```
nielslange@amun  ~/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty   master  npm run build

> twentytwenty@1.0.0 build /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
> npm run build:vendor-prefixes && npm run build:rtl


> twentytwenty@1.0.0 build:vendor-prefixes /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
> concurrently "npm run build:vendor-prefixes-style" "npm run build:vendor-prefixes-esb" "npm run build:vendor-prefixes-esc"

[1] 
[1] > twentytwenty@1.0.0 build:vendor-prefixes-esb /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[1] > postcss -r --no-map assets/css/editor-style-block.css 
[1] 
[0] 
[0] > twentytwenty@1.0.0 build:vendor-prefixes-style /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[0] > postcss -r --no-map style.css assets/css/editor-style-block.css assets/css/editor-style-classic.css
[0] 
[2] 
[2] > twentytwenty@1.0.0 build:vendor-prefixes-esc /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[2] > postcss -r --no-map assets/css/editor-style-classic.css
[2] 
[2] npm run build:vendor-prefixes-esc exited with code 0
[1] npm run build:vendor-prefixes-esb exited with code 0
[0] npm run build:vendor-prefixes-style exited with code 0

> twentytwenty@1.0.0 build:rtl /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
> concurrently "npm run build:rtl-style" "npm run build:rtl-esb" "npm run build:rtl-esc"

[1] 
[1] > twentytwenty@1.0.0 build:rtl-esb /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[1] > rtlcss assets/css/editor-style-block.css assets/css/editor-style-block-rtl.css
[1] 
[0] 
[0] > twentytwenty@1.0.0 build:rtl-style /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[0] > rtlcss style.css style-rtl.css
[0] 
[2] 
[2] > twentytwenty@1.0.0 build:rtl-esc /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty
[2] > rtlcss assets/css/editor-style-classic.css assets/css/editor-style-classic-rtl.css
[2] 
[2] Saving: /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty/assets/css/editor-style-classic-rtl.css
[1] Saving: /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty/assets/css/editor-style-block-rtl.css
[2] npm run build:rtl-esc exited with code 0
[0] Saving: /Users/nielslange/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty/style-rtl.css
[1] npm run build:rtl-esb exited with code 0
[0] npm run build:rtl-style exited with code 0
 nielslange@amun  ~/wp-local-docker-sites/wp-test/wordpress/wp-content/themes/twentytwenty   master  
```

Thus, the vendor prefixes seem to work as expected, at least on my machine.

**Comment by ianbelanger79** [comment]
Hmmm, I did run `npm install` first and @pattonwebz saw the same issues I did.

**Comment by ntwb** [comment]
This is the intended behaviour and appears to be working before the above cli output and my local testing.

The vendor-prefixes will now rarely ever be updated, they were added permanently to the CSS files, new vendor prefixes will only be added when a CSS change requires them to be added, updated, or removed.

**Comment by pattonwebz** [comment]
I actually had different output than the output above when I ran it before this issue was opened. I can't remember the exact error messages however I did confirm the very same issue on my end that Ian saw though. We both had the same problem.

CLI output that indicated NO REWRITES were done on the files. Some error about a no reader available or something along those lines. I don't quite remember the exact message.

There is a chance I ran the command without updating packages and used an older package.lock file though because running again now with a fresh test repo this runs fine for me with no errors :)
